src/components/navBar/index.tsx

```diff
@@ -1,17 +1,71 @@
-import Image from 'next/image';
+import Link from 'next/link';
+import useAuthenticated from '@/hooks/useAuthenticated';
+import {
+  LOGIN_URL,
+  USER_PROFILE_URL,
+  DEFAULT_AVATAR,
+  HOME_URL,
+  WELCOME_URL,
+  EVENTS_URL,
+  CRYPTO_URL,
+  MEMBERS_URL,
+  STATUS_URL,
+  GITHUB_LOGO,
+  RDS_LOGO
+} from '@/components/constants/url'
 import styles from '@/components/navBar/navBar.module.scss';
 
-const RDSLogo = '/RDSLogo.png';
 
-const NavBar = () => (
-  <nav className={styles.navBar}>
-    <a className={styles.logo} href="https://realdevsquad.com"><Image width="45px" height="45px" src={RDSLogo} alt="real-dev squad" /></a>
-    <a href="https://welcome.realdevsquad.com/">Welcome</a>
-    <a href="https://www.realdevsquad.com/events.html">Events</a>
-    <a href="https://members.realdevsquad.com/">Members</a>
-    <a href="https://crypto.realdevsquad.com/">Crypto</a>
-    <a className={styles.active} href="https://status.realdevsquad.com/">Status</a>
-  </nav>
-);
+const NavBar = () => {
+  const { userData, isLoggedIn } = useAuthenticated()
 
-export default NavBar;
+  return (
+    <nav className={styles.navBar}>
+      <div>
+        <a className={styles.logo} href={HOME_URL}>
+          <img width="45px" height="45px" src={RDS_LOGO} alt="real-dev squad" />
+        </a>
+        <a href={WELCOME_URL}>Welcome</a>
+        <a href={EVENTS_URL}>Events</a>
+        <a href={MEMBERS_URL}>Members</a>
+        <a href={CRYPTO_URL}>Crypto</a>
+        <a className={styles.active} href={STATUS_URL}>Status</a>
+      </div>
+      <div>
+        {!isLoggedIn ? (
+          <Link href={LOGIN_URL}>
+            <a className={styles.btnLogin}>
+              <button type="button" className={styles.btnLoginText}>
```

Comment:
Do we need an `<button>` inside an `<a>`?

Reply:
fixed